calico/getting-started/kubernetes/installation/config-options.md

```diff
@@ -291,19 +291,48 @@ Open the `install/kubernetes/istio-demo-auth.yaml` file in an
 editor, and locate the `istio-sidecar-injector` ConfigMap.  In the existing `istio-proxy` container, add a new `volumeMount`.
 
 ```
-{% include non-helm-manifests/istio-proxy-volume-mounts %}
+        - mountPath: /var/run/dikastes
+          name: dikastes-sock
 ```
 
 Add a new container to the template.
 
 ```
-{% include non-helm-manifests/dikastes-container %}
+      - name: dikastes
```

Comment:
I just inlined the contents of these files and removed the includes, so that we could get rid of all the _includes/non-helm-manifests